K8s 1.15 Helm integration tests failing

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
Since build [2343](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/master/2343/pipeline/61/), the 1.15 helm tests have been failing with the following errors about CRDs not found:
```
--- FAIL: TestCephHelmSuite (122.20s)
    ceph_installer.go:546: 
        	Error Trace:	ceph_installer.go:546
        	            				ceph_installer.go:530
        	            				ceph_base_deploy_test.go:188
        	            				ceph_base_deploy_test.go:178
        	            				ceph_base_deploy_test.go:161
        	            				ceph_helm_test.go:80
        	            				suite.go:102
        	            				ceph_helm_test.go:52
        	Error:      	Received unexpected error:
        	            	failed to get ceph cluster. the server could not find the requested resource (get cephclusters.ceph.rook.io test-cluster)
        	Test:       	TestCephHelmSuite
    ceph_installer.go:561: 
        	Error Trace:	ceph_installer.go:561
        	            				ceph_installer.go:530
        	            				ceph_base_deploy_test.go:188
        	            				ceph_base_deploy_test.go:178
        	            				ceph_base_deploy_test.go:161
        	            				ceph_helm_test.go:80
        	            				suite.go:102
        	            				ceph_helm_test.go:52
        	Error:      	Received unexpected error:
        	            	the server could not find the requested resource (get cephblockpools.ceph.rook.io)
        	Test:       	TestCephHelmSuite
        	Messages:   	failed to retrieve pool CRs
    ceph_installer.go:576: 
```

**Expected behavior:**
Integration tests should pass with v1beta1 extensions.

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary
* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

 To get logs, use `kubectl -n <namespace> logs <pod name>`
When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
Read [Github documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)):

--- a/cluster/charts/rook-ceph/templates/resources.yaml
+++ b/cluster/charts/rook-ceph/templates/resources.yaml
@@ -282,6 +282,18 @@ spec:
                         iteration:
                           type: integer
                           format: int32
+                security:
+                  type: object
+                  properties:
+                    kms:
+                      type: object
+                      properties:
+                        connectionDetails:
+                          type: object
+                          nullable: true
+                          x-kubernetes-preserve-unknown-fields: true
+                        tokenSecretName:
+                          type: string
                 annotations:
                   type: object
                   nullable: true
@@ -1272,7 +1284,9 @@ spec:
               x-kubernetes-preserve-unknown-fields: true
       subresources:
         status: {}
+---    
 {{- else }}
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
@@ -1438,6 +1452,7 @@ spec:
                   type: boolean
             placement: {}
             resources: {}
+            security: {}
             cleanupPolicy:
               properties:
                 confirmation:
@@ -1454,16 +1469,6 @@ spec:
                     iteration:
                       type: integer
                       format: int32
-            security:
-              properties:
-                kms:
-                  properties:
-                    connectionDetails:
-                      type: object
-                      nullable: true
-                      x-kubernetes-preserve-unknown-fields: true
-                    tokenSecretName:
-                      type: string
   additionalPrinterColumns:
     - name: DataDirHostPath
       type: string

--- a/cluster/examples/kubernetes/ceph/pre-k8s-1.16/crd.yaml
+++ b/cluster/examples/kubernetes/ceph/pre-k8s-1.16/crd.yaml
@@ -202,6 +202,7 @@ spec:
                     iteration:
                       type: integer
                       format: int32
+            security: {}
             placement: {}
             resources: {}
             healthCheck: {}

